client/src/client.ts

```diff
@@ -104,8 +104,9 @@ export class AngularLanguageClient implements vscode.Disposable {
   }
 }
 
-function registerNotificationHandlers(client: lsp.LanguageClient) {
-  client.onNotification(ProjectLoadingStart, () => {
+function registerNotificationHandlers(
+    client: lsp.LanguageClient, context: vscode.ExtensionContext) {
+  const d1 = client.onNotification(ProjectLoadingStart, () => {
```

Comment:
nit: d1 and d2 aren't great good variable names

Reply:
renamed to `disposable1` and `disposable2`.